packages/instrumenter/src/parsers/svelte-parser.ts

```diff
@@ -0,0 +1,100 @@
+import { BaseNode, Program } from 'estree';
+import { parse as svelteParse, walk } from 'svelte/compiler';
```

Comment:
```suggestion
```

As mentioned before, this import statement should not be done here. Loading the file will fail when `svelte` is not installed.